tasm-tour/src/components/ExhibitPage/ExhibitPage.js

```diff
@@ -40,22 +47,38 @@ export default function ExhibitPage({ exhibitID }) {
   }
 
   if (error) {
-    return <div className={errorStyle} style={{whiteSpace:"pre-wrap"}}> Error: {error}</div>;
+    return <div className={errorStyle} style={{ whiteSpace: "pre-wrap" }}> Error: {error}</div>;
   }
 
 
   if (!exhibit) {
     return (
-    <div className={errorStyle}>Exhibit not found. ID: {exhibitID}</div>
+      <div className={errorStyle}>Exhibit not found. ID: {exhibitID}</div>
     );
   }
 
+
+  console.log("Exhibit loaded successfully!\n" + exhibit);
+  let media = null;
+  switch (exhibit.mediaType) {
+    case 'model':
+      media = <ModelView modelID={exhibit.mediaLink} />;
+      break;
+    case 'video':
+      media = <VideoView videoPath={exhibit.mediaLink} />;
+      break;
+    case 'image':
+      media = <ImageView imagePath={exhibit.mediaLink} />;
+      break;
+    default:
+      media = null;
+  }
```

Comment:
Should this be moved to it's own component?

Reply:
I think it's fine where it is